code/modules/mob/living/silicon/robot/emote.dm

```diff
@@ -207,17 +224,31 @@
 			else
 				to_chat(src, "You are not security.")
 
-		/*
-		if ("fart")
-			var/list/robotfarts = list("makes a farting noise","vents excess methane","shakes violently, then vents methane.")
-			var/robofart = pick(robotfarts)
-			message = "<B>[src]</B> [robofart]."
-			m_type = VISIBLE
-
-		*/
+		if ("vent")
+			if(world.time-lastvent >= 30 SECONDS)
+				var/robofart = "[pick("whirrs","chuffs","clicks")] [pick("quietly","softly","sadly","excitedly","energetically","angrily","plaintively")] then [pick("vents","exhausts","releases")]"
+				for(var/mob/living/carbon/human/M in loc)
+					if(M != src && (M.lying || M.resting))
+						if(isvox(M))
```

Comment:
@Skullyton you mean making plasmaman ignite even in non-o2 atmos?

@gbasood Is there a way to have internals without having a mask on?